emuu-client/src/components/UserAuthentication/newRegister.js

```diff
@@ -32,70 +32,64 @@ function Register() {
     user_password: password,
   };
 
-
-
   const videoCollectionData = {};
 
-    const validatePassword= (pass)=>{
-    if(pass.length < 8){
-    document.querySelector("#password-err").innerHTML = "At least 8 characters";
-    document.querySelector("#password-err").style.color = "red";
-    return false;
-
+  const validatePassword = (pass) => {
+    if (pass.length < 8) {
+      document.querySelector("#password-err").innerHTML =
+        "At least 8 characters";
+      document.querySelector("#password-err").style.color = "red";
+      return false;
     }
 
     let uppercase = false;
     let specialChar = false;
-        for( let i = 0 ; i< pass.length; i++){
-    if(pass[i].charCodeAt(0) >= 65 && pass[i].charCodeAt(0)<= 90) uppercase = true;
+    for (let i = 0; i < pass.length; i++) {
+      if (pass[i].charCodeAt(0) >= 65 && pass[i].charCodeAt(0) <= 90)
+        uppercase = true;
     }
-    if(!uppercase) {
-                       document.querySelector("#password-err").innerHTML = "At least 1 uppercase letter"
-                        document.querySelector("#password-err").style.color = "red";
-                       return false;
-
-                       }
-    for( let i = 0 ; i< pass.length; i++){
-        if(pass[i].charCodeAt(0) >= 33 && pass[i].charCodeAt(0)<= 64) specialChar = true;
-        }
-       if(!specialChar) {
-                            document.querySelector("#password-err").innerHTML = "At least 1 special character"
-                             document.querySelector("#password-err").style.color = "red";
-                            return false;
-
-                            }
-
-       return true;
-
+    if (!uppercase) {
+      document.querySelector("#password-err").innerHTML =
+        "At least 1 uppercase letter";
+      document.querySelector("#password-err").style.color = "red";
+      return false;
+    }
+    for (let i = 0; i < pass.length; i++) {
+      if (pass[i].charCodeAt(0) >= 33 && pass[i].charCodeAt(0) <= 64)
+        specialChar = true;
+    }
+    if (!specialChar) {
+      document.querySelector("#password-err").innerHTML =
+        "At least 1 special character";
+      document.querySelector("#password-err").style.color = "red";
+      return false;
     }
 
+    return true;
+  };
 
-    function validateEmail(email)
-   {
-
+  function validateEmail(email) {
     let period = false;
     let at = false;
-    for( let i=0 ; i< email.length; i++){
-    if(email[i]== '@')at = true;
-    if(email[i]== '.')period = true;
+    for (let i = 0; i < email.length; i++) {
+      if (email[i] == "@") at = true;
+      if (email[i] == ".") period = true;
```

Comment:
Still the validation for email is not correct. Possible input that may bypass is a.@com

Reply:
I have made the email more secure in my commit